src/contracts/campaigns/BraidCampaign.sol

```diff
@@ -0,0 +1,195 @@
+pragma solidity ^0.4.4;
+
+
+import "utils/Owned.sol";
+import "verifiers/OwnedVerifier.sol";
+import "tokens/StandardToken.sol";
+
+/*
+A custom WeiFund compatable campaign for the Braid the Movie Ethereum campaign.
+*/
+
+contract BraidCampaign is Owned, StandardToken {
+  modifier pastFreezeDate {
+    // now must be greater than freeze date
+    if (block.number >= freezePeriod) {
+      _;
+    } else {
+      throw;
+    }
+  }
+
+  function transfer(address _to, uint256 _value) pastFreezeDate public returns (bool) {
+    return super.transfer(_to, _value);
+  }
+
+  function transferFrom(address _from, address _to, uint256 _value) pastFreezeDate public returns (bool success) {
+    return super.transferFrom(_from, _to, _value);
+  }
+
+  function approve(address _spender, uint256 _value) pastFreezeDate public returns (bool success) {
+    return super.approve(_spender, _value);
+  }
+
+  function () payable public {
+    // allow fallback function.. contribute as normal
+    contributeMsgValue(defaultAmounts);
+  }
+
+  function contributeMsgValue(uint256[] _amounts) payable public returns (uint256) {
+    // the total tokens amount to be issued to the sender
+    uint256 tokensOwed = msg.value / price;
```

Comment:
This no longer throws when there's remaining ether that doesn't by any tokens, so ether gets thrown away. The only valid options are throwing or refunding the extra ether. We can't just silently keep it without granting any tokens for it.